cmd/createredis.go

```diff
@@ -15,11 +15,31 @@
 package cmd
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
 
+	tcloudredis "github.com/rdxsl/tcloud-provisioner/tencent-cloud/redis"
 	"github.com/spf13/cobra"
 )
 
+var redisConfName string
+
+func redisParseConfig(tm *tcloudredis.TcloudRedis) error {
+	// Open our jsonFile
+	jsonFile, err := os.Open(redisConfName)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
We should talk tomorrow about how to handle printing errors and other output in a more consistent fashion